src/main/java/core/basesyntax/model/Transaction.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.model;
+
+public class Transaction {
+    private Operation operation;
+    private String name;
+    private int quantity;
+
+    public Transaction(String operationCode, String name, int quantity) {
+        this.operation = Operation.getByCode(operationCode);
+        this.name = name;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public enum Operation {
```

Comment:
I'd rather move this enum to separate file

Reply:
There was an example in README.

```
public class FruitTransaction {
  private Operation operation;
  private String fruit;
  private int quantity;

  // getters, setters, ...
  
  public enum Operation {
    BALANCE("b"),
    SUPPLY("s"),
    PURCHASE("p"),
    RETURN("r");

    private String code;

    Operation(String code) {
      this.code = code;
    }

    public String getCode() {
      return code;
    }
  }
}
```